.github/actions/run-tests/action.yml

```diff
@@ -0,0 +1,21 @@
+name: "Run tests"
+description: "Run tests and archive test results for subsequent publication"
+inputs:
+  command:
+    required: true
+    description: "The shell command to run tests"
+runs:
+  using: "composite"
+  steps:
+    - name: Run Tests
+      shell: bash
+      run: ${{ inputs.command }}
+
+    - name: Upload Test Results
+      uses: actions/upload-artifact@v2
+      if: always()
+      with:
+        name: Test Results ${{ github.job }}
+        path: |
+          **/test-results/**/*.xml
+          **/build/reports/jacoco/test/jacocoTestReport.xml
```

Comment:
jacoco is hardcoded -> what if we change the command and use another reporting tool like cobertura

Reply:
I am not sure this is called (see above).